admin/replication/LoadReplicationChanges

```diff
@@ -244,7 +244,7 @@ unless ($SCHEMA_SEQUENCE == $iSchemaSequence)
         $iSchemaSequence,
         ;
     print localtime() . " : You must upgrade your database in order to apply this replication packet\n";
-    exit 1;
+    exit 3;
```

Comment:
Same as in commit 4f53741a59307f597c2e7274464f6f8cc780d7dc can be useful here:

```suggestion
    # See NOTE-SEQM-1 in Script::JSONDump::Incremental
    exit 3;
```

Reply:
Thanks!